pkgs/applications/networking/cluster/kluctl/default.nix

```diff
@@ -1,32 +1,41 @@
-{ lib, buildGoModule, fetchFromGitHub, testers, kluctl }:
+{ lib, buildGoModule, fetchFromGitHub, testers, makeWrapper, python3, kluctl }:
 
 buildGoModule rec {
   pname = "kluctl";
-  version = "2.22.1";
+  version = "2.25.0";
 
   src = fetchFromGitHub {
     owner = "kluctl";
     repo = "kluctl";
     rev = "v${version}";
-    hash = "sha256-s7ADEWy3wx2hGeJzfXPVSBv+bAOoOQPsF75Sq02T/AI=";
+    hash = "sha256-WtTBkc9mop+bfMcVLI8k4Bqmift5JG9riF+QbDeiR9c=";
   };
 
   subPackages = [ "cmd" ];
 
-  vendorHash = "sha256-EEOVd15f1SK8InSIG+TuVwWibkf+ePJ5AGZpiMD+RaQ=";
+  vendorHash = "sha256-TckT39wQn4dclcYSfxootv1Lw5+iYxY6/wwdUc1+Z6s=";
 
   ldflags = [ "-s" "-w" "-X main.version=v${version}" ];
 
   # Depends on docker
   doCheck = false;
 
+  buildInputs = [
+    python3
```

Comment:
Why do we need this in buildInputs?

Reply:
I was under the impression that adding it to `buildInputs` was required as it is now a runtime dependency. It seems to still work if I leave out the `buildInputs` and specify `${lib.makeBinPath [ python3 ]}` later.
Can you confirm if that is the change you want?

Reply:
I would be interested in knowing why it _isn't_ needed there though.
Is just mentioning it in `makeBinPath` enough to make it available at runtime?

Reply:
> Is just mentioning it in makeBinPath enough to make it available at runtime?

yes